### DISCUSSION TITLE
Visualizer seek bar

### DISCUSSION THREAD

**Discussion by The-Ladle** [MissingCore/Music#58] [Ideas]
Basically just an option to set the seek bar as the same from the original design ![1695807112-958496-image.png](https://github.com/user-attachments/assets/cd17abd1-d70b-4595-a967-c984be3fbdcf)

**Comment by cyanChill** [comment]
I initially wanted to add this early on in the development process, but got stuck on trying to figure out how to implement this (as at the time, my React Native/mobile development knowledge is non-existent). I noted it down to revisit adding this in the future (guess I forgot to add this in the [Feature Road Map](https://github.com/MissingCore/Music/discussions/9)).

I think I have a better idea on how this could be implemented since I learned a bit about Kotlin and writing native modules (interface with native Android APIs in React Native) by creating the [`@missingcore/react-native-metadata-retriever`](https://github.com/MissingCore/react-native-metadata-retriever) package. However, this is more of a UI thing, I'll hold it off for now as it's a low priority.

**Comment by cyanChill** [comment]
This was implemented by @p-krystian and is now available as of [`v2.7.0-rc.2`](https://github.com/MissingCore/Music/releases/tag/v2.7.0-rc.2). To switch to this slider, go to `Settings > Experimental Features > Waveform Slider`.
